### PR TITLE
Cypress. Enabled check text with an ellipse rotation.

### PR DESCRIPTION
**Pull request by dvkruchinin**
Related on #4229 

### Motivation and context

Activation of the text occurrence check when the ellipse rotates.

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added a description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2022 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: tests/cypress/integration/actions_objects2/case_115_ellipse_shape_track_label.js [hunk 1]
```diff
@@ -78,8 +78,6 @@ context('Actions on ellipse.', () => {
                 (createEllipseTrackSwitchLabel.rightX + createEllipseTrackSwitchLabel.cx) / 2,
                 createEllipseTrackSwitchLabel.topY + 20,
                 '53.1',
-                false,
```
Comment (bsekachev) [inline tests/cypress/integration/actions_objects2/case_115_ellipse_shape_track_label.js:81]: Are you sure you want to remove ``pressShift`` argument?

Comment (dvkruchinin) [reply tests/cypress/integration/actions_objects2/case_115_ellipse_shape_track_label.js:81]: For this case, yes. The rotation check with shift occurs below.

Comment (bsekachev) [reply tests/cypress/integration/actions_objects2/case_115_ellipse_shape_track_label.js:81]: ``pressShift`` does not have a default value, and initialized by ``undefined`` in this case. 
Good practice is pass this argument or set a default value for the parameter.

Comment (dvkruchinin) [reply tests/cypress/integration/actions_objects2/case_115_ellipse_shape_track_label.js:81]: Set ```false``` as the default value for the parameter.

**Review by bsekachev — APPROVED** [review]

**Pull request merged**